src/axolotl/utils/trainer.py

```diff
@@ -579,6 +579,9 @@ def setup_trainer(cfg, train_dataset, eval_dataset, model, tokenizer, total_num_
         if cfg.bench_dataset:
             training_arguments_kwargs["bench_dataset"] = cfg.bench_dataset
 
+    if torch.__version__ >= "2" and sys.platform != "win32":
+        training_arguments_kwargs["torch_compile"] = True
```

Comment:
I think the conditional is probably unnecessary if HF is doing the work

But we will need to expose this to cfg because many things still fail trying to compile and train.

We may need the other compile options like backend exposed as cfg too

Reply:
thanks! updated